pyabc/sampler/redis_eps/cli.py

```diff
@@ -44,6 +48,7 @@ def handle(self, *args):
 def work_on_population(analysis_id: str,
                        t: int,
                        redis: StrictRedis,
+                       catch: bool,
```

Comment:
What does this catch variable do?

Reply:
that is mostly unrelated to the look-ahead thing, during tests I just wanted a way to make the workers fail directly when something goes wrong. the thing is, currently (without catch=False) when using the no-acceptance-delay version of look-ahead with some adaptive distance etc, there will never be thrown an error, but the sampler will just fail many times at at some point switch to non-look-ahead mode then.

this "catch" is however only semi-useful, because the calling main process never gets notified, one still has to manually check.